File: UnitTest/CS/DbManagerTest/OleDb/SqlServer.cs
using System;
using System.Data;

using NUnit.Framework;

using Rsdn.Framework.Data;

namespace CS.DbManagerTest.OleDb
{
	[TestFixture]
	public class SqlServer : CS.DbManagerTest.Test
	{
		public override string ConfigurationString 
		{
			get 
			{
				return "SqlServer.OleDb";
			}
		}

		public override string ParamText(string param)
		{
			return "?";
		}
	}
}

File: 1/UnitTest/VB/DbManagerTest/OleDb/SqlServer.vb
Imports System
Imports System.Data

Imports NUnit.Framework

Imports Rsdn.Framework.Data

Namespace DbManagerTest.OleDb

    <TestFixture()> _
    Public Class SqlServer
        Inherits VB.DbManagerTest.Test

        Public Overrides ReadOnly Property ConfigurationString() As String
            Get
                Return "SqlServer.OleDb"
            End Get
        End Property

        Public Overrides Function ParamText(ByVal param As String) As String
            Return "?"
        End Function

    End Class

End Namespace

